### PR TITLE
changed get_subject_sessions to work with multiple protocols

### PR DESCRIPTION
**Pull request by Hororohoruru**
A simple modification to the get_all_sessions function to make it able to work passing more than one protocol. The behavior when passing one protocol is unchanged. I think this might be useful for common uses of the function (i.e., fetching sessions for arbitrary sets of protocols for comparison, analysis, etc.)

### CODE REVIEW

File: ibc_public/utils_data.py [hunk 1]
```diff
@@ -41,17 +41,18 @@
                                      CONTRASTS['right label'][i]]
 
 
-def get_subject_session(protocol):
+def get_subject_session(*protocols):
```
Comment (bthirion) [inline ibc_public/utils_data.py]: The argument should be `protocols` not `*protocols`

File: ibc_public/utils_data.py [hunk 2]
```diff
@@ -41,17 +41,18 @@
...
     """ utility to get all (subject, session) for a given protocol"""
```
Comment (bthirion) [inline ibc_public/utils_data.py]: Please write a docstring

File: ibc_public/utils_data.py (hunk 1 above)
Comment (Hororohoruru) [reply ibc_public/utils_data.py]: Done. Is there any problem with the *args syntax? I find it more flexible since that way you are not forced to pass a list of one element when you want to pass only one argument

File: ibc_public/utils_data.py (hunk 2 above)
Comment (Hororohoruru) [reply ibc_public/utils_data.py]: Done

File: ibc_public/utils_data.py [hunk 3]
```diff
@@ -41,17 +41,35 @@
                                      CONTRASTS['right label'][i]]
 
 
-def get_subject_session(protocol):
-    """ utility to get all (subject, session) for a given protocol"""
+def get_subject_session(protocols: list) -> list:
```
Comment (bthirion) [inline ibc_public/utils_data.py]: `def get_subject_session(protocols):`
types are given in the dostring and only there. The code has to follow the numpy documentation conventions

https://numpydoc.readthedocs.io/en/latest/format.html

File: ibc_data/all_contrasts.tsv [hunk 4]
```diff
@@ -1,300 +1,300 @@
...
-archi_standard	left-right_button_press	Left vs right hand button press	right hand	left hand																																																											1	1	1
```
Comment (bthirion) [inline ibc_data/all_contrasts.tsv]: oops, this is an unrelated change !

Comment (bthirion) [reply ibc_data/all_contrasts.tsv]: Please fix this by separating your commits into different branches

File: ibc_public/utils_data.py (hunk 3 above)
Comment (Hororohoruru) [reply ibc_public/utils_data.py]: Done

File: ibc_data/all_contrasts.tsv (hunk 4 above)
Comment (Hororohoruru) [reply ibc_data/all_contrasts.tsv]: Done
